docs/userman/advanced_features/3rd party integration/databricks.md

```diff
@@ -0,0 +1,197 @@
+
+- _**table_name**_ represents the name of the Databricks table used to read and write
+    data pointed by the data nodes instantiated from the *Databricks Table* configuration.<br/>
+    It is used to populate the table name property of the entities (*Databricks Table* data nodes)
+    instantiated from the *Databricks Table* data node configuration. That means by default
+    all the entities (*Databricks Table* data nodes) instantiated from the same *Databricks Table*
+    configuration will inherit/share the same table name provided in the
+    table_name. To avoid this, the table name property of a *Databricks Table* data node entity
+    can be changed at runtime right after its instantiation.<br/>
+- _**profile**_ represents additional options of Databricks credential profile to be used when reading
+    and writing to the table.<br/>
+- _**cluster_id**_ represents additional options of the ID of Databricks cluster to be used when reading
+    and writing to the table.<br/>
+- _**exposed_type**_ indicates the data type returned when reading the data node (more
+    examples of reading from a Databricks Table data node with different *exposed_type* are available
+    in the [read/write a data node](data-node-usage.md#databricks-table) documentation):
+        - By default, *exposed_type* is "spark", and the data node reads the Databricks table
+            as a Spark DataFrame (`pyspark.sql.dataframe.DataFrame`) when executing the read method.
+        - If the *exposed_type* provided is "pandas", and the data node reads the Databricks table
+            as a Pandas DataFrame (`pandas.DataFrame`) when executing the read method.
+        - If the *exposed_type* provided is "numpy", the data node reads the Databricks table
+            as a NumPy array (`numpy.ndarray`) when executing the read method.
+        - If the *exposed_type* provided is "polars", and the data node reads the Databricks table
+            as a Polars DataFrame (`polars.DataFrame`) when executing the read method.
+
+To configure a Databricks SQL data node, use the `configure_databricks_table_data_node` method:
+
+```python linenums="1"
+{%
+include-markdown "./code-example/data-node-config/data-node-config_table_databricks.py"
+comments=false
+%}
+```
+
+In lines 3-9, we configure a basic Databricks Table data node with the id "historical_temperature".
+The data node will point to the Databricks table "hist_temp". Its *scope* is by
+default `SCENARIO`. The profile to be used is "default" profile with the cluster id is
+"0123-456789-dbscluster2". The exposed type of this data node will be the default value "spark".
+
+In lines 11-17, we configure another Databricks Table data node with the identifier "log_history".
+It uses the default `SCENARIO` scope again. The Databricks table name is "hist_log". The profile
+and cluster id are similar to the previous Databricks Table data node config. The *exposed_type*
+provided is "polars".
+
```

Comment:
Can we add one or two lines explaining the usage is the same as a normal data node. 
read/write/append, etc... Why not a short example.

What do you think?